### DISCUSSION TITLE
Do you know when are u updating this?

### DISCUSSION THREAD

**Discussion by Grillekkj** [KevinBytesTheDust/CVAmp#131] [General]

**Comment by jlplenio** [comment]
@Grillekkj are you referring to a specific feature or general updates?

**Reply by Grillekkj** [reply]
general updates, next release

**Reply by Grillekkj** [reply]
and the login/token feature too, kinda cool

**Reply by jlplenio** [reply]
I have a few ideas in the pipeline, but only find time on and off to pursue them:
- Almost done: A new GitHub Actions workflow to build macOS, Windows and Linux https://github.com/jlplenio/crude-twitch-viewer-bot/pull/129
- Soon: Modified current GUI with a new tab for logins and chat, and add Login and Chat capabilities.
- Later: Restructure the current GUI dynamically or replace with new GUI via https://github.com/zauberzeug/nicegui

**Reply by Grillekkj** [reply]
hmmmmm, that's cool, I'm excited for the login feature. Keep the great job!

**Reply by Grillekkj** [reply]
and i saw u commiting the github actions workflow
